recipes/sentry-native/all/conanfile.py

```diff
@@ -53,14 +53,16 @@ class SentryNativeConan(ConanFile):
     def _min_cppstd(self):
         if is_msvc(self):
             return "17"
-        if self.options.get_safe("backend") == "breakpad" and Version(self.version) >= "0.5.4":
+        if self.options.get_safe("backend") == "breakpad":
+            return 17
+        if self.options.get_safe("backend") == "crashpad" and Version(self.version) >= "0.7.0":
```

Comment:
It seems like it does not matter backend option here, by default it sets C++17 always.

https://github.com/getsentry/sentry-native/blob/0.7.0/CMakeLists.txt#L33

Reply:
Indeed. Done here: 34e39d589